apps/frontend/src/app/5_pages/PortfolioPage/components/AssetSection/components/AssetSectionActions/AssetSectionActions.tsx

```diff
@@ -54,7 +56,7 @@ export const AssetSectionActions: FC = () => {
             {t(translations.portfolioPage.assetSection.convert)}
           </Paragraph>
         </div>
-        {hasRbtcBalance && (
+        {hasRbtcBalance && account && (
```

Comment:
Put `hasRbtcBalance && account` as single variable to useMemo, so you could reuse it